lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2211,21 +2210,20 @@ export class UserAgentApplication {
     return this.config.auth.postLogoutRedirectUri;
   }
 
-  //#endregion
-
-  //#region String Util (Should be extracted to Utils.ts)
-
   /**
-   * Checks if the authorization endpoint URL contains query string parameters
-   * @ignore
-   * @hidden
+   * Used to get the current configuration of MSAL.js
    */
-  private urlContainsQueryStringParameter(name: string, url: string): boolean {
-    // regex to detect pattern of a ? or & followed by the name parameter and an equals character
-    const regex = new RegExp("[\\?&]" + name + "=");
-    return regex.test(url);
+  public getCurrentConfiguration(): Configuration {
+    if (!this.config) {
+      throw ClientConfigurationError.createNoSetConfigurationError();
+    }
+    return this.config;
```

Comment:
what does this enable?

Reply:
Enables us to see the state of the config of the msal object.